transformer_engine/pytorch/csrc/comm_gemm_overlap.h

```diff
@@ -302,12 +311,15 @@ struct UbufCommOverlap : torch::CustomClassHolder, UbufBase {
     if (B_scale_inverse.numel()) B_scale_inverse = B_scale_inverse[B_fp8_tensor];
 
     assert(pre_gelu_out.numel() == 0);
+    at::cuda::setCurrentCUDAStream(_stream_compute[0]);
```

Comment:
Why do we need that?

Reply:
Ok, nvm - I see that the te_gemm uses current Torch stream. We could probably change that.